core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -2484,7 +2504,8 @@ private RecordCursorFactory generateTableQuery(
                         readerTimestampIndex,
                         functionParser,
                         myMeta,
-                        executionContext
+                        executionContext,
+                        latestByHasOnlyOneColumn
```

Comment:
I guess name discrepancy between variable name and name of the parameter should suggest there is a flaw

variable name is `latestByHasOnlyOneColumn` parameter name is `canUseIndex`